### PR TITLE
Point jar.mn to chrome/ for newly added react files.

### PR DESCRIPTION
**Pull request by Mardak**
r?@Standard8

### CODE REVIEW

**Comment by mozilla-autolander-deprecated** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: add-on/jar.mn [hunk 1]
```diff
@@ -49,10 +49,10 @@
...
 #ifdef DEBUG
-  content/shared/vendor/react.js           (content/shared/vendor/react.js)
-  content/shared/vendor/react-dom.js       (content/shared/vendor/react-dom.js)
+  content/shared/vendor/react.js           (chrome/content/shared/vendor/react.js)
+  content/shared/vendor/react-dom.js       (chrome/content/shared/vendor/react-dom.js)
 #else
-  content/shared/vendor/react.js           (content/shared/vendor/react-prod.js)
-  content/shared/vendor/react-dom-prod.js  (content/shared/vendor/react-dom-prod.js)
+  content/shared/vendor/react.js           (chrome/content/shared/vendor/react-prod.js)
+  content/shared/vendor/react-dom-prod.js  (chrome/content/shared/vendor/react-dom-prod.js)
```
Comment (Mardak) [inline add-on/jar.mn]: Actually, this seems to be wrong in multiple ways. We want the file to be called `react-dom.js` always so `<script>` references work without `#ifdef`s?

Comment (Standard8) [inline add-on/jar.mn]: Yes we do!

**Comment by Standard8** [comment]
r=Standard8 with the additional rename.

**Comment by mozilla-autolander-deprecated** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by mozilla-autolander-deprecated** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Pull request merged**